sklearn/utils/estimator_checks.py

```diff
@@ -311,8 +315,8 @@ def _set_check_estimator_ids(obj):
         if not obj.keywords:
             return obj.func.__name__
 
-        kwstring = "".join(["{}={}".format(k, v)
-                            for k, v in obj.keywords.items()])
+        kwstring = ",".join(["{}={}".format(k, v)
```

Comment:
```suggestion
        kwstring = ", ".join(["{}={}".format(k, v)
```

Reply:
I have a slight preference for no space here, because this would be used for selecting tests when running `pytest` (on the command line)

```bash
test_estimators[AdaBoostClassifier()-check_classifiers_train(readonly_memmap=True,X_dtype=float32)]
```

or (with space)

```bash
test_estimators[AdaBoostClassifier()-check_classifiers_train(readonly_memmap=True, X_dtype=float32)]
```

Reply:
I did consider it initially, but I also find that without the space the naming of tests in pytest output is better.

Thanks for the reviews!